connectivity/drivers/ble/FEATURE_BLE/TARGET_Ambiq_Micro/CMakelists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(mbed-os-ble-apollo3 INTERFACE)
```

Comment:
```suggestion
add_library(mbed-os-ble-ambiq_micro INTERFACE)
```

Reply:
I believe the implementation is named apollo3 (target ambiq micro was used as it is a target name). If another target comes, apollo3 could be used as well.